eventDisplay/src/EventDisplay.cc

```diff
@@ -314,22 +314,12 @@ void EventDisplay::drawEvent(unsigned int id, bool resetCam) {
 
       std::unique_ptr<AbsKalmanFitter> fitter;
       switch (fitterId_) {
-        case SimpleKalman:
```

Comment:
But it seems some squareRootFormalism code (see below) is removed by this PR. Or is this only the remnants of some previous removals ?